README.md

```diff
@@ -9,15 +9,17 @@ Flutter widgets and themes implementing the current macOS design language.
   - [Contributing](#contributing)
   - [Resources](#resources)
 - [Layout](#layout)
-  - [Scaffold](#scaffold)
+  - [MacosScaffold](#macosscaffold)
 - [Buttons](#buttons)
-  - [Checkbox](#checkbox)
+  - [MacosCheckbox](#macoscheckbox)
   - [HelpButton](#helpbutton)
```

Comment:
MacosHelpButton, MacosRadioButton, MacosPushButton

Reply:
They don't overlap with any pre-existing widgets. Not needed.